src/test/java/de/retest/recheck/review/workers/LoadFilterWorkerTest.java

```diff
@@ -0,0 +1,39 @@
+package de.retest.recheck.review.workers;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+
+import de.retest.recheck.configuration.ProjectConfiguration;
+import de.retest.recheck.review.counter.NopCounter;
+
+public class LoadFilterWorkerTest {
+
+	@Test
+	void loading_without_ignore_file_should_fail( @TempDir final Path root ) throws Exception {
+		final LoadFilterWorker worker = new LoadFilterWorker( NopCounter.getInstance(), root.toFile() );
+		assertThrows( IllegalArgumentException.class, () -> worker.load() );
```

Comment:
We usually use AssertJ for assertions as they are—in our opinion—more flexible and readable.